crossdock/docker-compose.yml

```diff
@@ -21,12 +26,22 @@ services:
 
             - BEHAVIOR_TRACE=client,s1name,sampled,s2name,s2transport,s3name,s3transport
 
+            - AXIS_TESTDRIVER=test_driver
+            - AXIS_SERVICES=go,node,java,python
```

Comment:
not sure about this. The test driver is built in the `jaeger` repo, right? The only possible changes here would be the Go client, so why run the tests against other clients?